requirements-dev.txt

```diff
@@ -1,10 +1,8 @@
-autopep8
-attrs >=17.4
 black
+attrs >=17.4
 bump2version >=1.0.1
 check-manifest >=0.36
 doc8
-flake8
```

Comment:
I am not sure if we should remove `flake8`. Black doesn't check all the things flake8 checks. Maybe check this [pull request](https://github.com/compas-dev/compas/pull/1088/files) and try to match what we did in core, which is adding a `pyproject.toml` file to configure `black` and leave `flake8` in place with settings that match those of black so that they don't collide.

Reply:
@gonzalocasas thx! yep, about flake8 tom mentioned that to me too earlier, for the task.py and .toml would do the same with core.